### PR TITLE
Add /ess version subcommand

### PR DESCRIPTION
**Pull request by mdcfe**
Adds a `/ess version` command that gives people an easy way to find the relevant versions.

Looking at this class, there's a few things I noticed:
* By default, if no valid subcommand is supplied then it proceeds to reload the plugin. This probably isn't expected behaviour for most server owners.
* There are no permission checks (that I can see).
* Lots of `if (x.equalsIgnoreCase(y)) { ... } else if (x.equalsIgnoreCase(z)) { ... } else if ...`.

### CODE REVIEW

**Comment by SupaHam** [comment]
Looks good. Feel free to merge if you've tested.

**Comment by mdcfe** [comment]
I'm leaving this WIP and unmerged for the moment as:
1. I haven't had a chance to test it yet
2. I'm not sure exactly what format the Bukkit `Server` version will be yet, and what we'll need to do to get a useful version number for it.

**Comment by drtshock** [comment]
Looks good. Any reason not to pull?

**Review by drtshock — APPROVED** [review]

File: Essentials/src/com/earth2me/essentials/commands/Commandessentials.java
Comment (kashike) [inline Essentials/src/com/earth2me/essentials/commands/Commandessentials.java]: This doesn't look right.

**Review by kashike — CHANGES_REQUESTED** [review]

File: Essentials/src/com/earth2me/essentials/commands/Commandessentials.java
Comment (mdcfe) [reply Essentials/src/com/earth2me/essentials/commands/Commandessentials.java]: Good catch; it isn't right. It's been a while but I think the `!` belongs in the second statement.

Comment (kashike) [reply Essentials/src/com/earth2me/essentials/commands/Commandessentials.java]: Yes.

Comment (mdcfe) [reply Essentials/src/com/earth2me/essentials/commands/Commandessentials.java]: I think I enabled push access for maintainers, so if anyone wants to fix it feel free, as I'm busy for the next few weeks.

File: Essentials/src/com/earth2me/essentials/commands/Commandessentials.java
Comment (kashike) [inline Essentials/src/com/earth2me/essentials/commands/Commandessentials.java]: .equals

**Review by kashike — CHANGES_REQUESTED** [review]

File: Essentials/src/com/earth2me/essentials/commands/Commandessentials.java
Comment (mdcfe) [reply Essentials/src/com/earth2me/essentials/commands/Commandessentials.java]: Fixed, thanks for pointing that out

**Review by kashike — APPROVED** [review]

**Comment by mdcfe** [comment]
Example output on Paper build 1339 with a matching Antibuild, Chat and Protect and wrong EssentialsXSpawn version, as well as LP and Vault installed:


(This doesn't have the API version, but the actual PR branch does.)

**Review by mdcfe — COMMENTED** [review]
This is now ready to merge, unless anyone has any comments on how to improve this.

**Comment by LaxWasHere** [comment]
👍

**Pull request merged**
